package.json

```diff
@@ -34,6 +34,9 @@
     "dev": "turbo watch ci:test",
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
     "graph": "rimraf ./graph && mkdir graph && turbo run build --graph=graph/index.html",
+    "jsr:publish": "tsx ./scripts/jsr-publish.ts",
```

Comment:
Dry run completed successfully, but the actual `jsr publish` could still fail since it hasn't been executed yet.